webpack.config.js

```diff
@@ -8,11 +8,11 @@ dotenv.config();
 dotenv.config({path: '.env.default'});
 
 module.exports = (env = process.env.NODE_ENV || '') => ({
-  entry: './packages/node_modules/ciscospark',
+  entry: './packages/node_modules/webex',
   mode: env === 'production' ? 'production' : 'development',
   output: {
     filename: 'bundle.js',
-    library: 'ciscospark',
+    library: 'Webex',
```

Comment:
Lowercase?

Reply:
Nope, needs to be capitalized or else `webex` will be a global variable and will result in a `webex is already defined above` error since we use to do `var spark = ciscospark.init` and keeping it lowercase makes it `var webex = webex.init` so it needs to be `var webex = Webex.init`

/LongExplanation